src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -74,6 +77,21 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     };
   }
 
+  for (const navItem of taxonomies) {
+    const { entityBundle, name } = navItem;
+
+    if (name === 'Health Care') {
+      navItems[`${entityBundle}`] = navItem;
+    }
+  }
+
+  files['drupal/sidebar_navigation/index.html'] = {
```

Comment:
This looks like debugging info for the Health Care sidebar items. Is that correct?

Reply:
Yup correct, to print them to the console the same way y'all did for items on pages.